prbt_hardware_support/test/unittests/unittest_sto_modbus_adapter.cpp

```diff
@@ -225,7 +245,7 @@ TEST_F(PilzStoModbusAdapterTest, testClearMsg)
 
   EXPECT_CLEARANCE
 
-  pub_.publish(createDefaultStoModbusMsg(STO_CLEAR));
+      pub_.publish(createDefaultStoModbusMsg(STO_CLEAR));
```

Comment:
Seams like your IDE cannot deal with Macros without semicolons ...
Please fix the indentation in the whole file.